spec/lib/tasks/email_checkin_reminder_rake_spec.rb

```diff
@@ -4,9 +4,9 @@
 describe 'email_checkin_reminder' do
   include_context 'rake'
 
-  before(:all) { FactoryGirl.create(:app_config) }
-
   before(:each) do
+    @ac = mock_app_config(admin_email: 'admin@email.com',
```

Comment:
I assume we had to provide these attributes for the same reason as guessed in my [earlier comment](https://github.com/YaleSTC/reservations/pull/1566/files#r63644393)?